host-configs/rzgenie-toss_3_x86_64_ib-gcc@8.1.0.cmake

```diff
@@ -36,7 +36,7 @@ set(CMAKE_Fortran_COMPILER "/usr/tce/packages/gcc/gcc-8.1.0/bin/gfortran" CACHE
 ##############
 
 # Root directory for generated TPLs
-set(TPL_ROOT "/usr/WS1/axom/thirdparty_libs/builds/2019_08_29_16_57_11/gcc-8.1.0" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/thirdparty_libs/builds/2019_12_19_19_05_23/gcc-8.1.0" CACHE PATH "")
 
 # conduit from uberenv
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-master" CACHE PATH "")
```

Comment:
@bmhan12 -- I manually copied over the ``SPHINX_EXECUTABLE`` line (on line 61 below).

Is this necessary for our readthedocs-doxygen integration?
Or was it just something that you added while figuring it out?

If it's the former, we can modify the ``gcc@8`` configuration (within our uberenv/spack) to use ``+devtools``.
Otherwise, the only host-config that we're currently building the devtools for is ``rzgenie-clang@4``.